types/react-instantsearch-core/index.d.ts

```diff
@@ -798,7 +798,7 @@ export interface DynamicWidgetsExposed {
      * on the result of facetOrdering. This means that any child needs
      * to have either the “attribute” or “attributes” prop.
      */
-    children?: React.ReactChild;
+    children?: React.ReactElement | number | string;
```

Comment:
```suggestion
    children?: React.ReactElement;
```

only elements with the props `attribute` or `attributes`  are allowed